packages-integrations/svelte-scoped/test/cases/prose/OutputProd.svelte

```diff
@@ -1,4 +1,4 @@
-<div class="uno-prose" />
+<div class="usp-3ddsgd" />
```

Comment:
Since `uno-prose` is added to safelist, I think it should not be transformed and styles generated below started with `:is(.usp-3ddsgd)` will be redundant if `uno-prose` keeps as is

Reply:
`prose` is refacted into a `shortcut` now, and i found that svelte does some extra processing for `shortcut`, which i need to fix it.

Reply:
fixed in #4990